admin-manual/installation/upgrading.rst

```diff
@@ -54,15 +54,14 @@ purpose of the release and how best to install it.
 Make sure the dependencies are updated
 ======================================
 
-Since the 2.5 version, AtoM uses Elasticsearch 5.x. If you're upgrading from an
+As of version 2.9, AtoM uses Elasticsearch 6.8. If you're upgrading from an
 earlier version, please make sure to upgrade your Elasticsearch installation.
 You don't need to backup your index data as it will be recreated from the
 database later in the upgrade.
 
-You will also need to upgrade to Java 8 if the system is using a previous
-version, as this is a requirement for Elasticsearch 5.x.
+Elasticsearch is built using Java, and requires at least Java 8 in order to run.
```

Comment:
Do we know if the later openjdk LTS versions will work?

Reply:
hi @misilot , yes, later openjdk LTS versions should work but have not been tested thoroughly!